src/storage/src/hummock/state_store_v1.rs

```diff
@@ -518,7 +510,17 @@ impl StateStoreWrite for HummockStorageV1 {
     }
 }
 
-impl LocalStateStore for HummockStorageV1 {}
+impl LocalStateStore for HummockStorageV1 {
```

Comment:
shall we need some comments to explain that v1 is not part of the local_state_store and will be deprecated

Reply:
It is actually a requirement to impl LocalStateStore for HummockStorageV1 because state table relies on the underlying storage to impl LocalStateStore trait.